selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/DeleteOrganizationInListTest.java

```diff
@@ -19,73 +20,71 @@
 import com.google.inject.name.Named;
 import java.util.List;
-import org.eclipse.che.commons.lang.NameGenerator;
 import org.eclipse.che.multiuser.organization.shared.dto.OrganizationDto;
 import org.eclipse.che.selenium.core.client.TestOrganizationServiceClient;
 import org.eclipse.che.selenium.core.user.AdminTestUser;
-import org.eclipse.che.selenium.core.utils.WaitUtils;
 import org.eclipse.che.selenium.pageobject.dashboard.ConfirmDialog;
 import org.eclipse.che.selenium.pageobject.dashboard.Dashboard;
 import org.eclipse.che.selenium.pageobject.dashboard.NavigationBar;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationListPage;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /**
- * Test validates organization creation and actions on it in the list of organizations.
+ * Test validates deleting organizations from the list of organizations.
  *
  * @author Ann Shumilova
  */
 public class DeleteOrganizationInListTest {
-  private static final Logger LOG = LoggerFactory.getLogger(DeleteOrganizationInListTest.class);
+  private static final String ORGANIZATION_NAME = generate("organization", 7);
 
   private List<OrganizationDto> organizations;
   private OrganizationDto organization;
-
-  @Inject private OrganizationListPage organizationListPage;
-  @Inject private NavigationBar navigationBar;
-  @Inject private ConfirmDialog confirmDialog;
-  @Inject private Dashboard dashboard;
+  private int organizationsCount;
 
   @Inject
   @Named("admin")
   private TestOrganizationServiceClient testOrganizationServiceClient;
 
+  @Inject private OrganizationListPage organizationListPage;
+  @Inject private NavigationBar navigationBar;
+  @Inject private ConfirmDialog confirmDialog;
   @Inject private AdminTestUser adminTestUser;
+  @Inject private Dashboard dashboard;
 
   @BeforeClass
   public void setUp() throws Exception {
-    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
-
-    String organizationName = NameGenerator.generate("organization", 5);
-    organization = testOrganizationServiceClient.create(organizationName);
+    organization = testOrganizationServiceClient.create(ORGANIZATION_NAME);
+    testOrganizationServiceClient.create(generate("organization", 7));
+    testOrganizationServiceClient.create(generate("organization", 7));
+    testOrganizationServiceClient.create(generate("organization", 7));
     organizations = testOrganizationServiceClient.getAll();
+
+    organizationsCount = organizations.size();
+    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(organization.getId());
+    for (OrganizationDto organization : organizations)
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.